src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -417,3 +417,8 @@ This flag is **deprecated** and **has no effect**.
 Rustdoc only supports Rust source code and Markdown input formats. If the
 file ends in `.md` or `.markdown`, `rustdoc` treats it as a Markdown file.
 Otherwise, it assumes that the input file is Rust.
+
+## `--nocapture`
+
+When this flag is used with `--test`, the output (stdout and stderr) of your tests won't be
+captured anymore. This is especially useful for debugging your tests!
```

Comment:
I think it would be good to clarify what "capture" means, in case people don't know:

```suggestion
captured by rustdoc. Instead, the output will be directed to your terminal,
as if you had run the test executable manually. This is especially useful
for debugging your tests!
```